Lib/sqlite3/dbapi2.py

```diff
@@ -85,3 +89,11 @@ def convert_timestamp(val):
 # Clean up namespace
 
 del(register_adapters_and_converters)
+
+def __getattr__(name):
+    if name in _deprecated_names:
+        from warnings import warn
+        warn(f"{name} is deprecated and will be removed in Python 3.14",
```

Comment:
Style (linebreak after local import)
```suggestion

        warn(f"{name} is deprecated and will be removed in Python 3.14",
```